codec/number.js

```diff
@@ -14,8 +16,8 @@ exports.encode = function (number) {
 
     var splitScientificNotation = number.toExponential().split('e');
     var exponent = Number(splitScientificNotation[1]) + 500;
-    var mantissa = splitScientificNotation[0] + (splitScientificNotation[0].indexOf('.') === -1 ? '.' : '') + '0'.repeat(20);
-    var encoded = 'E' + padStart(String(exponent), 3) + 'M' + String(mantissa);
+    var mantissa = splitScientificNotation[0] + (splitScientificNotation[0].indexOf('.') === -1 ? '.' : '')//+ '0'.repeat(20);
```

Comment:
Since the zeros are not appended anymore, there is no need to add a '.' after the mantissa if it is a round number

Reply:
updated